weave-js/src/core/ops/domain/run.ts

```diff
@@ -689,3 +689,30 @@ export const opRunRuntime = makeRunOp({
     return run.computeSeconds;
   },
 });
+
+export const opStrRunlink = OpKinds.makeBasicOp({
+  name: 'str-runlink',
+  argTypes: {
+    entity_name: {
+      type: 'union' as const,
+      members: ['none' as const, 'string' as const],
+    },
+    project_name: {
+      type: 'union' as const,
+      members: ['none' as const, 'string' as const],
+    },
+    name: {
+      type: 'union' as const,
+      members: ['none' as const, 'string' as const],
+    },
+  },
+  description: `Returns a link to the ${docType('run')} overview`,
+  argDescriptions: {run: runArgDescriptions},
+  returnValueDescription: `The link to the ${docType('run')} overview`,
+  returnType: inputTypes => 'link',
+  resolver: ({entity_name, project_name, name}) => {
+    return (
```

Comment:
Can you please replace this implementation with code that throws an error saying it is a python-only op that